### PR TITLE
CNV-8875: New release note - stable channel update

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-8875](https://issues.redhat.com/browse/CNV-8875)
For 4.9 release notes

Preview: https://deploy-preview-36349--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-9-release-notes

### CODE REVIEW

**Review by sbulage — APPROVED** [review]
LGTM :+1:

File: virt/virt-4-9-release-notes.adoc [hunk 1]
```diff
@@ -46,6 +46,9 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 //CNV-9540 High performance windows
 
+//CNV-8875
+* If your {VirtProductName} Operator subscription used any update channel other than *stable*, it is now automatically subscribed to the *stable* channel. The *stable* channel subscribes you to minor version updates and ensures that your {VirtProductName} version is compatible with your {product-title} version.
```
Comment (sbulage) [inline virt/virt-4-9-release-notes.adoc:50]: Can you please change {VirtProductName} and {product-title} to bold text .?

Comment (sabrinajess) [reply virt/virt-4-9-release-notes.adoc:50]: @sbulage Per our style guide, we use bold text when referring to a specific UI item, and not to highlight or emphasize.

Comment (sjhala-ccs) [inline virt/virt-4-9-release-notes.adoc:50]: Suggestion for the second sentence - "This single update channel delivers z-stream and minor version updates and ensures that your {VirtProductName} and {product-title} versions are compatible."

**Review by sjhala-ccs — COMMENTED** [review]
Just a suggestion for you to consider, otherwise LGTM

File: virt/virt-4-9-release-notes.adoc (hunk 1 above)
Comment (sabrinajess) [reply virt/virt-4-9-release-notes.adoc:50]: I like the sound of that better. Will revise.

**Pull request merged**